### PR TITLE
feat: Validate tracks have at least one segment. Add tests.

### PR DESCRIPTION
**Pull request by DuckyCB**
## Description
- Validate tracks have at least one segment
- Return object with details about the track validity
- Add tests

## Requirements Checklist
- [ ] Unit Tests updated or fixed
- [ ] Docs/guides updated

### CODE REVIEW

File: lib/src/cmaf/ham/services/validateTracks.ts [hunk 1]
```diff
@@ -1,20 +1,53 @@
...
+		// Validate same duration
 		if (!duration) {
 			duration = track.duration;
 		}
 		if (track.duration !== duration) {
-			isValid = false;
+			description.sameDuration = false;
+		}
+		// Validate one or more segments
+		if (!track.segments?.length) {
```
Comment (NoeliaBentancor) [inline lib/src/cmaf/ham/services/validateTracks.ts:40]: Why is this necessary?

**Review by mpollingerQualabs — APPROVED** [review]

File: lib/src/cmaf/ham/services/validateTracks.ts (hunk 1 above)
Comment (DuckyCB) [reply lib/src/cmaf/ham/services/validateTracks.ts:40]: To validate there is one or more segments

**Review by NoeliaBentancor — APPROVED** [review]

**Review by matiasrb97 — APPROVED** [review]
